lms/djangoapps/courseware/date_summary.py

```diff
@@ -20,27 +19,46 @@
 class DateSummary(object):
     """Base class for all date summary blocks."""
 
-    # The CSS class of this summary. Indicates the type of information
-    # this summary block contains, and its urgency.
-    css_class = ''
+    @property
+    def css_class(self):
+        """
+        The CSS class of this summary. Indicates the type of information
+        this summary block contains, and its urgency.
+        """
+        return ''
 
-    # The title of this summary.
-    title = ''
+    @property
+    def title(self):
+        """The title of this summary."""
+        return ''
 
-    # The detail text displayed by this summary.
-    description = ''
+    @property
+    def description(self):
+        """The detail text displayed by this summary."""
+        return ''
 
-    # This summary's date.
-    date = None
+    @property
+    def date(self):
+        """This summary's date."""
+        return None
 
-    # The format to display this date in. By default, displays like Jan 01, 2015.
-    date_format = '%b %d, %Y'
+    @property
+    def date_format(self):
+        """
+        The format to display this date in. By default, displays like Jan
+        01, 2015.
+        """
+        return '%b %d, %Y'
```

Comment:
Future improvement: move this to a [setting](https://docs.djangoproject.com/en/1.8/ref/settings/#date-format) so that it is easily localized for open source instances.